ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/account/AndroidAccount.kt

```diff
@@ -59,10 +62,14 @@ internal fun WrapAccount(
 
     val walletSnapshot = walletViewModel.walletSnapshot.collectAsStateWithLifecycle().value
 
+    val isHideBalances = homeViewModel.isHideBalances.collectAsStateWithLifecycle().value ?: false
```

Comment:
What if we collected this `val isHideBalances = homeViewModel.isHideBalances.collectAsStateWithLifecycle().value ?: false` in final composable only. This way we would dodge having to delegate the flag everywhere 🤔

Reply:
Collecting the viewmodel within related compostables would make testing the compostables harder. This way, we're able to prepare a simple UI test that verifies proper view layer setup based on its inputs.